src/table/utils/__tests__/selections-util.spec.ts

```diff
@@ -134,17 +148,23 @@ describe('selections-utils', () => {
       expect(state.api.begin).toHaveBeenCalledTimes(1);
       expect(state.api.select).toHaveBeenCalledWith({ method: 'selectHyperCubeCells', params });
       expect(state.api.cancel).not.toHaveBeenCalled();
-      expect(action.payload.announce).toHaveBeenCalledTimes(1);
+      expect(action?.payload?.announce).toHaveBeenCalledTimes(1);
```

Comment:
payload is optional since some actions don't have a payload, need optional chaining here